docs/outlook/onmessagefromchanged-onappointmentfromchanged-events.md

```diff
@@ -50,20 +47,19 @@ The following tables list client-server combinations that support the `OnMessage
 
 ## Prerequisites
 
-To preview the `OnMessageFromChanged` and `OnAppointmentFromChanged` events, set up your preferred client accordingly.
+To test the walkthrough, you must have at least two Exchange accounts.
 
-- For Outlook on Windows, install Version 2212 (Build 15919.10000) or later. Once installed, join the [Microsoft 365 Insider program](https://insider.microsoft365.com/join/Windows) and select the **Beta Channel** option to access Office beta builds.
-- For Outlook on Mac, install Version 16.69.116 or later.
-- For Outlook on the web, ensure that the Targeted release option is set up on your Microsoft 365 tenant. To learn more, see the "Targeted release" section of [Set up the Standard or Targeted release options](/microsoft-365/admin/manage/release-options-in-office-365#targeted-release).
-
-To test the walkthrough, you must also have at least two Exchange accounts.
+To preview the `OnMessageFromChanged` and `OnAppointmentFromChanged` events in Outlook on Mac, install Version 16.69.116 or later.
 
 ## Set up your environment
 
 Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator), which creates an add-in project with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
 
 ## Configure the manifest
 
+> [!NOTE]
+> The `OnMessageFromChanged` and `OnAppointmentFromChanged` events aren't yet supported for the [Unified Microsoft 365 manifest (preview)](../develop/json-manifest-overview.md).
```

Comment:
Please confirm if this is the name or "Unified manifest for Microsoft 365 (preview)"

Reply:
Thanks for catching this! Verified that "Unified manifest for Microsoft 365 (preview)" is the new name (see https://github.com/OfficeDev/office-js-docs-pr/pull/4014) and made the necessary changes.